src/apps/vpn/translations/strings.yaml

```diff
@@ -112,6 +112,12 @@ serversView:
   serverCityWithNoConnection:
     value: "%1. No connectivity"
     comment: Button label that reads out a city name whose connection is unavailable. Argument is the city name.
+  recommendedEmptyTitle: 
+    value: Turn off VPN to see list
```

Comment:
Was this shortened to fit in one line? It sound strange without articles, e.g. `Turn off VPN to see the list`.

Reply:
Thats certainly what it looks like. The strings are also seem redundant. What do you think of:

Title: Recommended servers unavailable
Description: To see recommended server locations, please turn the VPN off